File: 2014/j3.py
# code by savir singh

n = int(input())
antonia = 100
david = 100

for i in range(n):
    a = input().split(" ")
    if int(a[0]) < int(a[1]):
        antonia -= int(a[1])
    elif int(a[0]) == int(a[1]):
        pass
    else:
        david -= int(a[0])

print(antonia)
print(david)

File: 2014/j3.cpp
// code by savir singh

#include <stdio.h>

int main()
{
    int numRounds;
    int aScore = 100, dScore = 100;
    int dice1, dice2;
    scanf("%d", &numRounds);

    for (int i = 0; i < numRounds; i++)
    {
        scanf("%d %d", &dice1, &dice2);
        if (dice1 == dice2)
        {
            continue;
        }
        else if (dice1 > dice2)
        {
            dScore -= dice1;
        }
        else
        {
            aScore -= dice2;
        }
    }

    printf("%d\n%d", aScore, dScore);
}
